language-server/src/server.js

```diff
@@ -32,6 +33,7 @@ setMetaSchemaOutputFormat(DETAILED);
 setShouldValidateSchema(false);
 
 const isSchema = RegExp.prototype.test.bind(/(?:\.|\/|^)schema\.json$/);
+const availableSchemas = getDialectIds();
```

Comment:
This can't be global. If someone adds a custom meta-schema in their workspace, that needs to show up in the list as well. I hadn't thought of this until now, but if someone removes their custom meta-schema from their workspace, it should be removed from the list. We might need to add another function in `@hyperjump/json-schema` to remove a dialect that can be called when a custom meta-schema is removed from the workspace.